.github/workflows/workspace.yml

```diff
@@ -0,0 +1,81 @@
+name: Workspace
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+
+permissions:
+  contents: read
+
+jobs:
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dtolnay/rust-toolchain@master
+      with:
+        toolchain: nightly-2024-06-11
+        components: clippy,rust-src
+    - name: Linux (linux_android_with_fallback.rs)
+      run: cargo clippy --target x86_64-unknown-linux-gnu
+    - name: Linux (linux_android.rs)
+      run: cargo clippy --target x86_64-unknown-linux-gnu --features linux_disable_fallback
+    - name: macOS (getentropy.rs)
+      run: cargo clippy -Zbuild-std --target x86_64-apple-darwin
```

Comment:
Here and elsewhere (except for the `js` implementation), we should just do `build-std=core`, as we don't need to build all of the standard library just for clippy.